.github/workflows/odh_notebook_controller_integration_test.yaml

```
@@ -82,12 +82,14 @@ jobs:
         run: |
           set -x
 
-          cd components/odh-notebook-controller/config/default
+          cd components/odh-notebook-controller/config/base
           kubectl create ns opendatahub
 
-          export CURRENT_NOTEBOOK_IMG=quay.io/opendatahub/odh-notebook-controller:latest
-          export PR_NOTEBOOK_IMG=localhost/${{env.IMG}}:${{env.TAG}}
-          kustomize edit set image ${CURRENT_NOTEBOOK_IMG}=${PR_NOTEBOOK_IMG}
+          # export CURRENT_NOTEBOOK_IMG=quay.io/opendatahub/odh-notebook-controller:latest
+          # export PR_NOTEBOOK_IMG=localhost/${{env.IMG}}:${{env.TAG}}
+          # kustomize edit set image ${CURRENT_NOTEBOOK_IMG}=${PR_NOTEBOOK_IMG}
+          sed -i "s/quay.io\/opendatahub\/odh-notebook-controller:stable/localhost\/${{env.IMG}}:${{env.TAG}}/g" params.env
```

you have different tag in your params file here, they need to match; or, even better, I would not use sed, but echo >, to rewrite the file entirely with the new img

```suggestion
          echo "odh-kf-notebook-controller-image=localhost\/${{env.IMG}}:${{env.TAG}}" > params.env
```

---

not sure about this, however i will push your suggestion to check it

---

it's never good when we have ci checks written in such a way that one can't run them on local machine easily ;(

---

https://github.com/opendatahub-io/kubeflow/actions/runs/9955370866/job/27502971667?pr=364 

```
Run set -x
+ cd components/odh-notebook-controller/config/base
+ kubectl create ns opendatahub
namespace/opendatahub created
+ echo 'odh-kf-notebook-controller-image=localhost\/odh-notebook-controller:integration-test'
+ oc apply -f -
+ cat
configmap/notebook-controller-culler-config created
+ cat
+ kubectl apply -f -
secret/odh-notebook-controller-webhook-cert created
+ kustomize build .
+ sed 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g'
+ kubectl apply -f -
Error: field specified in var '{odh-notebook-controller-image ~G_v1_ConfigMap {data.odh-notebook-controller-image}}' not found in corresponding resource
error: no objects passed to apply
Error: Process completed with exit code 1.
```

---

guess you were right to doubt this, `\/` is escape for the sed command, without sed it will be just `/`

```suggestion
          echo "odh-kf-notebook-controller-image=localhost/${{env.IMG}}:${{env.TAG}}" > params.env
```